Readme.md

```diff
@@ -37,7 +37,6 @@ Links
 
 ### Community
 
-* [GDevelop forums](https://forum.gdevelop-app.com) and [Discord chat](https://discord.gg/rjdYHvj).
```

Comment:
?

Reply:
Can you revert this change? :)

Reply:
i am not sure how that was deleted. i would fix that quickly.